### PR TITLE
Create sieve_of_eratosthenes.java

### PR DESCRIPTION
**Pull request by enigma0503**

### CODE REVIEW

File: sieve_of_eratosthenes.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+	for(int i=2;(i*i)<=n;i++)                //start from 2 as 2 is the first prime number.
+	{
+		if(arr[i]==1)                         
+		{
+		for(int j=i*2;j<=n;j+=i)               /* replace the value at all indexes that are multiples of j with 0 
+		{                                         ultimately we wiil be left with the numbers that are not multiples of any other number.*/
+			arr[j]=0;
+		}
+	}
+  }
```
Comment (ghost) [inline sieve_of_eratosthenes.java]: Please remove this extra parenthesis.

**Review by ghost — CHANGES_REQUESTED** [review]
Parenthesis at line number 27 is extra. Please review that.

**Comment by enigma0503** [comment]
no............it is perfectly ok
Try to run the code first.

File: sieve_of_eratosthenes.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+	for(int i=0;i<=n;i++)                 //initialize all the indexes with 1.
+	{
+		arr[i]=1;
+	}
+	for(int i=2;(i*i)<=n;i++)                //start from 2 as 2 is the first prime number.
+	{
+		if(arr[i]==1)                         
+		{
+		for(int j=i*2;j<=n;j+=i)               /* replace the value at all indexes that are multiples of j with 0 
+		{                                         ultimately we wiil be left with the numbers that are not multiples of any other number.*/
```
Comment (khanchi97) [inline sieve_of_eratosthenes.java]: you have commented this "curly brace" too, either use inline comment or move the brace to new line.

**Comment by khanchi97** [comment]
Closed this PR as it has no file changed.

**Pull request closed without merging**